wom/src/main/scala/wdl4s/wdl/WdlExpression.scala

```diff
@@ -33,11 +33,10 @@ object WdlExpression {
     def isMapLiteral: Boolean = ast.getName == "MapLiteral"
     def isObjectLiteral: Boolean = ast.getName == "ObjectLiteral"
     def isArrayOrMapLookup: Boolean = ast.getName == "ArrayOrMapLookup"
-    def params = ast.getAttribute("params").asInstanceOf[AstList].asScala.toVector
+    def params: Vector[AstNode] = Option(ast.getAttribute("params")).map(_.asInstanceOf[AstList].asScala.toVector).getOrElse(Vector.empty)
     def name = ast.getAttribute("name").asInstanceOf[Terminal].getSourceString
-    def isFunctionCallWithOneParameter = ast.isFunctionCall && ast.params.size == 1
-    def isFunctionCallWithOneFileParameter = isFunctionCallWithOneParameter && WdlFunctionsWithSingleFileParameter.contains(ast.functionName)
-    def isGlobFunctionCall = isFunctionCallWithOneParameter && "glob".equals(ast.functionName)
+    def isFunctionCallWithFirstParameterBeingFile = ast.isFunctionCall && ast.params.nonEmpty && WdlFunctionsWithFirstParameterBeingFile.contains(ast.functionName)
```

Comment:
I find this name awkward but have no better suggestion. If no one else has a problem with this ignore my comment